lib/maropost/contact.rb

```diff
@@ -35,14 +35,14 @@ def initialize(opts = {})
       self.email = data[:email]
       self.phone_number = data[:phone]
       self.cell_phone_number = data[:cell_phone_number]
-      self.do_not_contact = data.fetch(:do_not_contact, false)
+      self.allow_emails = data.fetch(:allow_emails) { !DoNotMailList.exists?(self) }
```

Comment:
this doesn't need to match up with anything in the maripost api right?

Reply:
That's correct. Basically this attribute has different logic when compared to the maropost "lists".

The value of this attribute is directly determined by whether the the user is on the do not mail list or not.